images/linux/scripts/installers/containers.sh

```diff
@@ -15,4 +17,8 @@ apt-get -qq -y install ${install_packages[@]}
 mkdir -p /etc/containers
 echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
 
+if isUbuntu20 ; then
+    rm /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
```

Comment:
we need to document it somehow, not just remove from the list

Reply:
Agree, May be we can collect the list of repos during image generation to some separate file like `echo "repo1" >> /image-generation/repos.txt` and attach this file to readme later?

Reply:
Btw, why we remove repo only on Ubuntu 20?